.env

```diff
@@ -0,0 +1,2 @@
+MONGO_URI=localhost
+MONGO_PORT=27017
```

Comment:
This file shouldn't be committed. We should include a .env.sample file instead.

Reply:
is it ok if it stays in the history? this is the same contents as would go in a .env.sample

Reply:
History is okay. Just need to rename the file.